source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,216 @@
+
+- Community  - Our entire team is active within our public instance of Mattermost, plus you’ll have the support of one of the best open source communities around
+- Documentation - We link to many parts of our `administrator  documentation <https://docs.mattermost.com/guides/administrator.html>`_ in this guide but encourage you to take a look at `all our documentation <https://docs.mattermost.com>`_
+- Enterprise Support  - If you’re an Enterprise Edition subscriber, you may open a support ticket in the Enterprise Edition Support portal
+
+.. Recommendation::
+    Open our public instance of Mattermost right now in a separate browser tab and create a user. Join channel ..::TODO Specific Guide Support Channel?::.. to not only experience Mattermost right away but to ask if getting stuck at any point in this guide.
+
+================================
+Before You Begin Your Deployment
+================================
+
+********************************
+What Makes Mattermost Different?
+********************************
+
+Mattermost is a high-trust messaging platform for enterprise use. As enterprises come in all shapes and sizes, Mattermost has been built to provide an optimal experience for all customers. 
+
+Taking this a step further, the emergence of ChatOps means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of integrations, extensions, and customization.
+
+Here’s just a sample of available integrations our customers are deploying:
+
+- Confluence
+- Docker
+- GitHub / GitLab
+- Jira
+- Outlook
+- Trello
+
+Taking this further, because Mattermost is an Open Source application, it can be completely customized, in every way, to suit your organizational needs.
+
+***************
+Further Reading
+***************
+
+Mattermost Licensing
+--------------------
+
+Mattermost Enterprise Edition comes with two licensing options: Enterprise E10 and Enterprise E20 which can be tested with a `Free 30-Day Trial <https://mattermost.com/trial/>`_. We tailored the E10 licence to suit smaller organizations with less need for automation and compliance, where our E20 license will provide you with features like ADFS SAML 2.0 support, automated compliance export and team specific permissions. You can find a full comparison `here <https://mattermost.com/pricing-feature-comparison>`_.
+
+
+.. Note::
+    Reach out to our Sales team by contacting sales@mattermost.com to upgrade your license at any time.
+
+Mattermost Server Environment
+-----------------------------
+
+At its core, Mattermost is an open source, hybrid-cloud alternative to proprietary SaaS messaging for teams. It is designed to increase the agility, efficiency and innovation in high trust organizations while keeping data and operations under IT control. Mattermost ensures you can own your data. An important step is to determine the environment for the Mattermost server.
```

Comment:
```suggestion
At its core, Mattermost is an open source, hybrid-cloud alternative to proprietary SaaS messaging for teams. It is designed to increase the agility, efficiency and innovation in high trust organizations while keeping data and operations under IT control. Mattermost ensures you can own your data. An important step is to determine the environment for the Mattermost server.
```
Suggestion: Confirm that this is a current description.

Reply:
This was taken from [here](https://docs.mattermost.com/overview/product.html#about-the-mattermost-open-source-project). AFAIK We're in a status of transition but it still holds true, or doesn't it? WDYT @justinegeffen

Reply:
I'll defer to @wiersgallak on this one as that page hasn't been updated recently and an update may be needed.